deafrica/monitoring/latency_check.py

```diff
@@ -177,7 +180,6 @@ def cli(
         satellite=satellite,
         latency=latency,
         notification_slack_url=slack_url,
-        Bucket= Bucket,
+        Bucket=Bucket,
         Prefix=Prefix,
-        

```

Comment:
@Zintle95 please remember to add `Bucket` and `Prefix` as arguments to the actual function definition (line 79)